modules/user/inputs.tf

```
@@ -0,0 +1,35 @@
+variable "user" {
+  default     = null
+  description = "The user to create"
```

```suggestion
  description = "[DEPRECATED] Object containing all the variables below."
```

To make it clearer not to use this?